lib/doorkeeper/oauth/forbidden_token_response.rb

```diff
@@ -3,6 +3,8 @@
 module Doorkeeper
   module OAuth
     class ForbiddenTokenResponse < ErrorResponse
+      MISSING_SCOPE_I18N = "doorkeeper.errors.messages.forbidden_token.missing_scope"
```

Comment:
I don't sure we have to allocate a constant for this, let's move it below

Reply:
OK, I will make that change, also will resolve conflict with CHANGELOG.

Reply:
@nbulaj I have adjusted the code to not use a constant, and rebased over latest `main` to resolve the conflict in CHANGELOG